### ISSUE TITLE
test time query

### ISSUE THREAD

**Issue by sahilm1992** [boathit/deepst#6]
Could you please tell us how do you get traffic T.C at test-time?

Is there a separation of queries used for testing and used for getting T.C at test-time?

**Comment by boathit** [comment]
We process all the data in the same way and partition them into training/validation/test dataset.

**Comment by sahilm1992** [comment]
Ok, so during testing, if I have a test query Source-B-C-D-E-F-G-Destination.
Is it also utilized for getting T.C at test time?

**Comment by boathit** [comment]
It will not be used for creating T.C for itself but will be used for the other trajectories in next sliding window.

**Comment by sahilm1992** [comment]
Ok. So lets say at test time I got a slot 8-8:30 A.M.
There are 100 trips( Trip1, Trip2,.... Trip100) that start betwen 8- 8:30 A.M.

While testing any arbitary trip lets say Trip50,,,, will T1, T2... T49, T51, T52... T100 be used to get T.C ?

**Comment by boathit** [comment]
No. The trips falling into 8:00-8:30 a.m will only be used for creating T.C for trips in 8:30-8:50 a.m.

**Comment by sahilm1992** [comment]
Sure, Thanks..

**Comment by sahilm1992** [comment]
So in that case, the traffic snapshot is slightly old?
